account_banking_pain_base/models/payment_line.py

```diff
@@ -20,33 +20,23 @@
 #
 ##############################################################################
 
-from openerp.osv import orm, fields
+from openerp import models, fields
 
 
-class PaymentLine(orm.Model):
+class PaymentLine(models.Model):
     _inherit = 'payment.line'
 
-    def _get_struct_communication_types(self, cr, uid, context=None):
+    def _get_struct_communication_types(self):
```

Comment:
We can even remove this method and "hard-code" it, because now we can make _selection_add_ in overrides.